build.cake

```diff
@@ -669,7 +674,11 @@ Task("SRGen")
                 var xlfFile = xlfDoc.Files.Single();
 
                 // load a language template
-                var templateFileLocation = System.IO.Path.Combine(locTemplateDir, System.IO.Path.GetFileName(docName) + ".template");
+                var templateDirToUse = locTemplateDir;
+                if(parentDir.Equals("external")){
```

Comment:
Do we actually need this check? The loc folder is the same root folder for both it seems

Reply:
Yeah you're right about that, it's completely unnecessary